.github/workflows/testnet.yml

```diff
@@ -58,18 +72,40 @@ jobs:
           docker push $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG
           echo "::set-output name=image::$ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG"
 
-      - name: Fill in the new image ID in the Amazon ECS task definition
+      - name: Fill in the new image ID / us-west-2 - gateway
         id: task-def
         uses: aws-actions/amazon-ecs-render-task-definition@v1
         with:
-          task-definition: pocket-gateway/tasks/staging/ecs-task-testnet.json
+          task-definition: pocket-gateway/tasks/testnet/ecs-task-testnet.json
           container-name: gateway-testnet
           image: ${{ steps.build-image.outputs.image }}
+          # Needs to be set at container level to be read by the dd agent
+          environment-variables: |
+            DD_RUNTIME_METRICS_ENABLED=true
+            DD_PROFILING_ENABLED=true
+            DD_SERVICE=/us-west-2/ecs/gateway
```

Comment:
I think `testnet` should be in here otherwise it will get mixed in with production logs